engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/AnimationScreen.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.nui.layers.ingame.metrics;
+
+import com.google.common.base.Optional;
```

Comment:
You might have accidentally gotten this `Optional` imported instead of `java.util.Optional` - that's why later on you end up with `java.util.Optional<Prefab> prefab` where if you take off the `java.util.` it results in an error.